### PR TITLE
Crazy mode

### PR DESCRIPTION
**Pull request by 9728627**
I have added the crazy mode in the game. there will more jelly fish and it will more challenging for users to win the game.

### CODE REVIEW

File: src/edu/ucsb/cs56/projects/games/fish_animation/Menu.java [hunk 1]
```diff
@@ -305,12 +305,14 @@ public void setDifficulty() {
 		Medium = new JButton("MEDIUM");
 		Hard = new JButton("HARD");
 		Resume = new JButton("RESUME");
+		Crazy = new JButton ("Crazy");
```
Comment (xavierholt) [inline src/edu/ucsb/cs56/projects/games/fish_animation/Menu.java]: This should be capitalized to match the other buttons.

**Comment by xavierholt** [comment]
I don't think there was an issue for this, but I guess I could give you 20 XP?  You didn't really do anything except copy and paste...

**Comment by 9728627** [comment]
So what should I do? Do I need to go to the diffBG branch and make the differences there?

**Comment by xavierholt** [comment]
The diffBG branch is already merged.  You should make sure this branch has the changes from master (`git pull origin master`) and then make your changes on this branch.

**Comment by 9728627** [comment]
ok get it. I just wonder how can I get 100 xp? I chose the error because it was 100xp. It has dropped from 200xp to 100xp. Do I need to add more features to it to make it seem worth 100xp?

**Comment by xavierholt** [comment]
You're not going to get 100 XP for this feature.  Make it work with the different backgrounds feature and I'll give you 25 XP.

Don't try to add things to this feature to get more points.  You'll just make it messy and I'll reject all of your changes.

I apologize for making you think you could get more points for this.  I dropped the initial point estimate significantly because the feature didn't seem worth the original estimate.  And that was just a hunch - now that I've seen the implementation I have to drop it further.  I can't give you any more points because this is just copy-and-paste.

If you're worried about getting to 500 by midnight, don't be.  I don't expect _any_ of my groups to have 500 XP merged by tonight.  I'm pretty strict on code review, and that slows things down.  As long as you have 500 XP in progress / in PRs you're in good shape.  That's what we're looking at for the lab five grades.  You will have to get those merged eventually, though!

**Comment by 9728627** [comment]
Get it. thanks for your time

**Comment by xavierholt** [comment]
Fixed by https://github.com/UCSB-CS56-Projects/cs56-games-fish-animation/pull/95#pullrequestreview-78280281 which has all the changes from this PR plus bugfixes.

**Pull request closed without merging**